modules/worker-utils/src/lib/worker-api/get-worker-url.ts

```diff
@@ -41,7 +41,11 @@ export function getWorkerURL(worker: WorkerObject, options: WorkerOptions = {}):
   // If URL is test, generate local loaders.gl url
   // @ts-ignore _workerType
   if (options._workerType === 'test') {
-    url = `modules/${worker.module}/dist/${workerFile}`;
+    if (isBrowser) {
```

Comment:
OK. Maybe we could add some comments here and/or in the function header comment.

Reply:
Done